### DISCUSSION TITLE
Is it possible to call elisp from inside sqlite?

### DISCUSSION THREAD

**Discussion by dmgerman** [skeeto/emacsql#124] [Support] [answered]
First of all, thank you for this wonderful package. I have used it extensively (directly or via other packages that use it) using sqlite .

I have a question: is it possible to call emacs-lisp (or other languages) functions from inside sqlite once it is running inside emacs?

Something like create_function in python (calling python from functions inside sqlite queries): https://docs.python.org/3/library/sqlite3.html

thank you,

**Comment by tarsius** [comment, marked as answer]
No, that is not supported by EmacSQL. It also does not appear to be supported by the builtin `sqlite.c` and the `sqlite3-api.c` of the `sqlite3` package (which in turn are used by EmacSQL's builtin and module back-ends.

When you need such advanced functionality, it would IMO be better to use one of these implementations directly. But until one of them has actually implemented support for that, that's a moot point.

I also don't know if this is even feasible (I am fare from a SQL[ite] expert). It does sound interesting. You might want to look into implementing support for that in Emacs yourself; that would be a useful contribution.
